### ISSUE TITLE
[fill-in-the-blanks] addons21\1933645497\handler.py:92:model is deprecated: please use 'note_type'

### ISSUE THREAD

**Issue by khonkhortisan** [ssricardo/anki-plugins#117]
```
addons21\1933645497\handler.py:92:model is deprecated: please use 'note_type'
```
https://github.com/ssricardo/anki-plugins/blob/594ed8adbfe12f4bb08d75af9516c8413ab70ba0/fill-the-blanks/src/handler.py#L92

 - OS: Windows 10
 - Anki Version 2.1.55 (35431c59)⁩

**Comment by ssricardo** [comment]
Hi! Thanks for reporting.
I'll take a look on that

**Comment by ssricardo** [comment]
Addressed in version 2.7 (10.01.2023)
